org.eclipse.jdt.ui/ui/org/eclipse/jdt/ui/text/java/correction/CUCorrectionProposal.java

```diff
@@ -253,7 +253,7 @@ private boolean performValidateEdit(ICompilationUnit unit) {
 	 * @throws CoreException if the creation of the text change failed
 	 */
 	protected synchronized TextChange createTextChange() throws CoreException {
-		if( useDelegateToCreateTextChange() ) {
+		if (useDelegateToCreateTextChange()) {
```

Comment:
Again a whitespace only change. Please consider submitting a separate PR that does formatting changes where needed.